docker/images/github-cli/Dockerfile

```
@@ -0,0 +1,21 @@
+ARG base_image=bitnami/minideb:latest
+
+FROM ${base_image}
+
+RUN apt-get update -qq && apt-get upgrade -y && apt-get install -y ca-certificates git
```

With minideb we no longer need to run `apt-get upgrade`, because Bitnami keeps the base image up to date for us. This way we stay secure and avoid bloating the image with old versions of packages.

```suggestion
RUN apt-get update -qq && apt-get install -y ca-certificates git
```